example/threejs-video-ts.html

```diff
@@ -183,26 +180,22 @@
                 var image_data = ctx.getImageData(0, 0, width, height);
 
                 if (image_data.data) {
-                    
                     vec = ar.update(grayscale(image_data.data));
                     console.log("Marker found is: ", vec.get(0));
                     conf = ar.getConfidence();
-                   
                     console.log("Marker confidence is: ", conf * 100, "%");
                     var matrix = ar.getModelViewMatrix();
-
                     console.log("Marker pose matrix is:", matrix);
-                    //artk.arglCameraViewRHf(matrix, glRHMatrix)
```

Comment:
It's not needed to convert the model matrix to Rught Hand GL matrix, but i think this function can be useful.